features/git-hack/main_branch_rebase_tracking_branch_conflict/with_open_changes.feature

```diff
@@ -13,7 +13,14 @@ Feature: git hack: resolving conflicts between main branch and its tracking bran
       |        | local    | conflicting local commit  | conflicting_file | local content  |
     And I am on the "existing_feature" branch
     And I have an uncommitted file with name: "uncommitted" and content: "stuff"
-    When I run `git hack new_feature` while allowing errors
+    When I run `git hack new_feature`
+    Then it errors and the output ends with
+      """
```

Comment:
+1 on this. Feature specs should verify that the tool works, not that everything looks properly indented and spaced out. If we start doing that, then the specs become a whole lot more brittle.